wrangler-core/src/main/java/io/cdap/wrangler/executor/RecipePipelineExecutor.java

```diff
@@ -112,6 +114,12 @@ public List<Row> execute(List<Row> rows) throws RecipeException {
           context.getTransientStore().reset(TransientVariableScope.LOCAL);
         }
 
+        // Get input schema from transientStore
+        // TODO check if possible to access workspace id somehow for prefix - worst case change set variable dir
+        Schema schema = context.getEnvironment() != null &&
+          context.getEnvironment().equals(ExecutorContext.Environment.SERVICE) ?
+          context.getTransientStore().get("ws_schema") : null;
```

Comment:
nit: should we call it as output schema for readability here and other places including updated schema(to become updated output schema)?

Reply:
Done